README.md

```diff
@@ -49,12 +47,37 @@ Flyway will not run and you will also need to provide values for each schema (1-
  CSR_REGION1_PASSWORD
 ```
 
+```
+ jdbc:oracle:thin:@localhost:1521/schema
+ username
+ password
+```
+
+You will also need to port forward to the dev database possibly using the port-forward-pod.
```

Comment:
possibly using the port-forward-pod -> this can be done by creating a port-forward-pod